Dockerfile

```diff
@@ -40,10 +40,10 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN \
   apt-get -qq update && \
-  apt-get -y -qq install apt-utils && \
-  apt-get -y -qq dist-upgrade && \
-  apt-get -y -q install postfix && \
-  apt-get -y -q --no-install-recommends install \
+  apt-get -qq install apt-utils 2>/dev/null && \
```

Comment:
`2>/dev/nulll`

What's the rational behind this? It will hide error output (again)?

Reply:
With `apt-utils`, the error message will always be the same: `debconf: delaying package configuration, since apt-utils is not installed` 😆 This is an inherently useless message, I figured. Hence, the error output from this command can be hidden.